go/src/koding/klient/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:14.04
```

Comment:
Currently `klient.deb` is built with ubuntu 14.04 debootstrap, golang:1.7 is based on debian wheezy. It is a safe choice, there's no sense in testing how`klient/build` works when built by other distros, especially when long-term plan is to remove it and don't use .deb for cloud-init to support distros other than debian based.